sign-up/src/SignUp.js

```diff
@@ -0,0 +1,77 @@
+import React from "react";
+import { useEffect } from "react";
+import { useState } from "react";
+import Box from "@mui/material/Box";
+import Typography from "@mui/material/Typography";
+import Grid from "@mui/material/Grid";
+import TextField from "@mui/material/TextField";
+import Button from "@mui/material/Button";
+import Link from "@mui/material/Link";
+import { createTheme, ThemeProvider } from "@mui/material/styles";
+import useMediaQuery from "@mui/material/useMediaQuery";
```

Comment:
please check if there are extra imports here, it is possible that we might not require everything that we have imported. This should be a practice that should always be followed. There are a bunch of plugins that VS code that can help you debug this. Please take a look into this. If you are not able to find the plugins for this, please reach out to us.